src/components/pages/collectRecordFormPages/BenthicPhotoQuadratForm/BenthicPhotoQuadratObservationTable.js

```diff
@@ -60,79 +60,30 @@ const StyledColgroup = styled('colgroup')`
 `
 
 const BenthicPhotoQuadratObservationTable = ({
-  areObservationsInputsDirty,
   areValidationsShowing,
-  benthicAttributeOptions,
+  benthicAttributeSelectOptions,
   choices,
   collectRecord,
-  observationsReducer,
-  openNewObservationModal,
-  persistUnsavedObservationsUtilities,
   ignoreObservationValidations,
+  observationsReducer,
   resetObservationValidations,
   setAreObservationsInputsDirty,
+  setIsNewBenthicAttributeModalOpen,
+  setObservationIdToAddNewBenthicAttributeTo,
+  testId,
 }) => {
-  const [isObservationReducerInitialized, setIsObservationReducerInitialized] = useState(false)
   const [autoFocusAllowed, setAutoFocusAllowed] = useState(false)
   const [observationsState, observationsDispatch] = observationsReducer
 
-  const {
-    persistUnsavedFormData: persistUnsavedObservationsData,
-    getPersistedUnsavedFormData: getPersistedUnsavedObservationsData,
-  } = persistUnsavedObservationsUtilities
-
-  const _ensureUnsavedObservationsArePersisted = useEffect(() => {
-    if (areObservationsInputsDirty) {
-      persistUnsavedObservationsData(observationsState)
-    }
-  }, [areObservationsInputsDirty, observationsState, persistUnsavedObservationsData])
-
-  const handleAddEmptyInitialObservation = useCallback(() => {
-    setAreObservationsInputsDirty(true)
-
-    observationsDispatch({ type: 'addObservation' })
-  }, [observationsDispatch, setAreObservationsInputsDirty])
-
   const handleAddObservation = () => {
     setAreObservationsInputsDirty(true)
```

Comment:
currently stuck as those values are still not persisting if I toggle online/offline. Checked to see that `setAreObservationsInputsDirty(true)` are being set for every update same as lit/pit.